tests/php/Centreon/Domain/ConfigurationLoader/YamlConfigurationLoaderTest.php

```diff
@@ -22,14 +22,13 @@
 
 namespace Centreon\Domain\ConfigurationLoader;
 
+use Symfony\Component\Filesystem\Exception\FileNotFoundException;
```

Comment:
To delete if we use the standard exception in the YamlConfigurationLoader object